### PR TITLE
[BUG][Connector-V2][Jdbc] support postgresql json type 

### PR DESCRIPTION
**Pull request by jackyyyyyssss**
## Purpose of this pull request
issues #5103 

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by ic4y** [comment]
We need to add json and jsonb data types in the pg e2e tests.

**Comment by jackyyyyyssss** [comment]
> We need to add json and jsonb data types in the pg e2e tests.

ok

**Comment by jackyyyyyssss** [comment]
> We need to add json and jsonb data types in the pg e2e tests.

@ic4y PTAL and rebuild ci thks

**Comment by ic4y** [comment]
Currently, the e2e tests have an issue with image pull failure. It's being addressed in https://github.com/apache/seatunnel/pull/5201.

**Review by zhilinli123 — CHANGES_REQUESTED** [review]
Add a postgresql connector document type mapping

**Comment by zhilinli123** [comment]
https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/PostgreSql.md
https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/PostgreSQL.md

**Comment by jackyyyyyssss** [comment]
> https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/PostgreSql.md https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/PostgreSQL.md

ok

File: docs/en/connector-v2/sink/PostgreSql.md [hunk 1]
```diff
@@ -125,6 +125,7 @@ transform {
 
 sink {
     jdbc {
+       # if you would have json or jsonb type insert please add jdbc url stringtype=unspecified option
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/PostgreSql.md]: If the user uses JSON-related types, tell it how to configure the url information, please prompt in the url Description in Options
image

Comment (jackyyyyyssss) [reply docs/en/connector-v2/sink/PostgreSql.md]: done @zhilinli123 PTAL

**Review by zhilinli123 — APPROVED** [review]

**Comment by zhilinli123** [comment]
PTAL: @ic4y

**Review by hailin0 — APPROVED** [review]

**Review by zhilinli123 — APPROVED** [review]

**Review by ic4y — APPROVED** [review]
Awesome! Thanks for your great contribution! 🎉

**Pull request merged**